projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/vdkTags.ts

```diff
@@ -98,17 +104,23 @@ export const trackVdkTags = (
           vdkCellIndices.push(cellIndex);
         cellIndex++;
       }
-
+      // this case covers the use case when the notebook is loaded for the first time
+      if (vdkCellIndices.length == 0) {
+        vdkCellIndices = await getVdkCellIndices(
```

Comment:
nit: `if (not vdkCellIndices)`? (I see the codacy warning of 0)

Reply:
done